lib/pre-build.js

```diff
@@ -95,7 +95,7 @@ function findNodeGyp() {
   if (process.env.npm_config_node_gyp) {
     try {
       gypPath = process.env.npm_config_node_gyp
-      if (fs.existsSync(gypPath)) {
+      if (fs.accessSync(gypPath)) {
```

Comment:
this ticket was super vague but it was suggested to just remove this entire try/catch.  But I would just leave untouched.  This was a straight ripoff of this logic https://github.com/mapbox/node-pre-gyp/blob/e0b3b6/lib/util/compile.js#L18-L55. I don't know enough of the user case where a user would set `npm_config_node_gyp` but this supports it 🤷🏻‍♂️

Reply:
Can you clarify? Should I remove the entire try/catch or just revert my changes? `existsSync` is deprecated and has been superseded by `accessSync` so if we are going to keep this block around, we might as well use modern methods, right?

Reply:
true, that's fair